flexmeasures/data/models/reporting/profit.py

```diff
@@ -32,13 +32,17 @@ class ProfitReporter(Reporter):
         (+) gains
         (-) losses
 
+    This sign convention can be adapted to your needs:
+        - The power/energy convention can be inverted by setting the sensor attribute `consumption_is_positive` to True.
+        - The output (gains/losses) sign can be inverted by setting the reporter config attribute `is_profit` to False.
```

Comment:
From https://github.com/FlexMeasures/flexmeasures/pull/825#discussion_r1300301762: using `loss_is_positive` should make this easier to grasp, and with better symmetry to `consumption_is_positive`.

Reply:
Good point!